components/blitz/resources/omero/Constants.ice

```diff
@@ -158,6 +158,7 @@ module omero {
         const string NSINSIGHTTAGSET = "openmicroscopy.org/omero/insight/tagset";
         const string NSINSIGHTRATING = "openmicroscopy.org/omero/insight/rating";
         const string NSMOVIE = "openmicroscopy.org/omero/movie";
+        const string NSCLIENTMAPANNOTATION = "openmicroscopy.org/omero/client/mapAnnotation";
```

Comment:
The Idea was to have one map annotation that the user will be editing. One per user.
Other could be generated during acquisition are not editable in clients.
This was discussed in MapAnnotation mockup